streams/src/main/java/org/apache/kafka/streams/processor/internals/DefaultKafkaClientSupplier.java

```diff
@@ -48,4 +48,9 @@ public AdminClient getAdminClient(final Map<String, Object> config) {
     public Consumer<byte[], byte[]> getRestoreConsumer(Map<String, Object> config) {
         return new KafkaConsumer<>(config, new ByteArrayDeserializer(), new ByteArrayDeserializer());
     }
+
+    @Override
+    public Consumer<byte[], byte[]> getGlobalConsumer(Map<String, Object> config) {
```

Comment:
nit: add `final` to parameter (we we also add `final` for the other methods above?)

Reply:
I took a look of `KafkaConsumer` class. We couldn't set the parameter to final since it is still modifying the content when initializing at line 609.

Reply:
We can still set `final` -- note, that `final` implies that the parameter `config` cannot be reassigned to a different `Map` -- the `Map` itself is still mutable though and thus it can be altered within `KafkaConsumer` later on. (Adding `final` is more a question of style here.)